framework/include/userobject/UserObject.h

```diff
@@ -201,3 +204,12 @@ UserObject::gatherProxyValueMax(T1 & value, T2 & proxy)
   _communicator.maxloc(value, rank);
   _communicator.broadcast(proxy, rank);
 }
+
+template <typename T1, typename T2>
+void
+UserObject::gatherProxyValueMin(T1 & value, T2 & proxy)
+{
+  unsigned int rank;
+  _communicator.minloc(value, rank);
+  _communicator.broadcast(proxy, rank);
+}
```

Comment:
```suggestion
template <typename T1, typename T2>
void
UserObject::gatherProxyValueMin(const T1 & value, T2 & proxy)
{
  processor_id_type rank;
  _communicator.minloc(value, rank);
  _communicator.broadcast(proxy, rank);
}
```

I _think_ you can make `value` const, but you should definitely use `processor_id_type` for the rank. I know this is just copying another implementation - if you wouldn't mind making this change for the max method as well and add a docstring for both.

Reply:
I can use `processor_id_type`, but I can't make the value const - I get lots of errors like this:

```
/home/anovak/moose/scripts/../libmesh/installed/include/timpi/parallel_implementation.h: In instantiation of ‘void TIMPI::Communicator::maxloc(T&, unsigned int&) const [with T = const double]’:
/home/anovak/moose/framework/include/userobject/UserObject.h:216:3:   required from ‘void UserObject::gatherProxyValueMax(const T1&, T2&) [with T1 = double; T2 = double]’
/home/anovak/moose/framework/src/postprocessors/ElementExtremeValue.C:98:47:   required from here
/home/anovak/moose/scripts/../libmesh/installed/include/timpi/parallel_implementation.h:2409:22: error: use of deleted function ‘TIMPI::DataPlusInt<const double>::DataPlusInt()’
 2409 |       DataPlusInt<T> data_in;
```